pkg/cache/cache.go

```diff
@@ -49,7 +49,33 @@ func (tc *TimeCache) Add(item string) bool {
 		return false
```

Comment:
Maybe we should update the given item with a current `time.Now()` value?

Reply:
Oh yeah, that would mean we have to move the item to the front too. Not sure if it's worth the hassle.